server/server.go

```diff
@@ -385,25 +385,57 @@ func (r *rodeServer) initialize(ctx context.Context) error {
 	return nil
 }
 
-func (r *rodeServer) ListOccurrences(ctx context.Context, occurrenceRequest *pb.ListOccurrencesRequest) (*pb.ListOccurrencesResponse, error) {
-	log := r.logger.Named("ListOccurrences")
-	log.Debug("received request", zap.Any("ListOccurrencesRequest", occurrenceRequest))
+func (r *rodeServer) ListVersionedResourceOccurrences(ctx context.Context, request *pb.ListVersionedResourceOccurrencesRequest) (*pb.ListVersionedResourceOccurrencesResponse, error) {
+	log := r.logger.Named("ListVersionedResourceOccurrences")
+	log.Debug("received request", zap.Any("ListVersionedResourceOccurrencesRequest", request))
 
-	request := &grafeas_proto.ListOccurrencesRequest{
-		Parent:    rodeProjectSlug,
-		Filter:    occurrenceRequest.Filter,
-		PageToken: occurrenceRequest.PageToken,
-		PageSize:  occurrenceRequest.PageSize,
+	resourceUri := request.ResourceUri
+	if resourceUri == "" {
+		return nil, createErrorWithCode(log, "invalid request", errors.New("must set resource_uri"), codes.InvalidArgument)
+	}
+
+	log.Debug("listing build occurrences")
+	buildOccurrences, err := r.grafeasCommon.ListOccurrences(ctx, &grafeas_proto.ListOccurrencesRequest{
```

Comment:
Not handling pagination since here there it's not likely that there would be more than a few build occurrences